python/tests/test_docker.py

```diff
@@ -107,4 +109,4 @@ def test_docker():
     assert test_status, 'test failed'
 
 if __name__ == '__main__':
-    test_docker()
+    pytest.main(sys.argv)
```

Comment:
Instead of calling the test methods directly you can just let pytest take care of them and then you don't have to worry about having one test entrypoint per file.